### PR TITLE
Store the origin country-code

### PR DESCRIPTION
**Pull request by ludeeus**
Docs: https://github.com/home-assistant/home-assistant.io/pull/17232

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Review by bramkragten — APPROVED** [review]

File: site/src/data.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export type AnalyticsData = Record<string, Analytics>;
 
 export interface Analytics {
   active_installations: number;
+  country: Record<string, number>;
```
Comment (frenck) [inline site/src/data.ts:15]: Countries?

**Review by frenck — APPROVED** [review]

File: worker/src/handlers/schedule.ts [hunk 2]
```diff
@@ -60,6 +60,7 @@ const generateCurrentDataset = (
   const installation_types = { os: 0, container: 0, core: 0, supervised: 0 };
   const integrations: Record<string, number> = {};
   const addons: Record<string, number> = {};
+  const country: Record<string, number> = {};
```
Comment (bramkragten) [inline worker/src/handlers/schedule.ts:63]:
```suggestion
  const countries: Record<string, number> = {};
```

**Review by bramkragten — APPROVED** [review]

**Comment by ludeeus** [comment]
Good to go, but docs should be synced to `rc` before this is merged

**Comment by frenck** [comment]
Docs synced.

**Pull request merged**